trajopt/src/problem_description.cpp

```diff
@@ -1763,4 +1763,50 @@ void TotalTimeTermInfo::hatch(TrajOptProb& prob)
   }
 }
 
+void AvoidSingularityTermInfo::fromJson(ProblemConstructionInfo&, const Json::Value&)
+{
+  CONSOLE_BRIDGE_logWarn("Not implemented yet");
```

Comment:
I recommend making this an error and adding an assert(false) to make it easy to track this down in debug mode using gdb.